from uel.errors.uelexception import UELException


class UELRuntimeError(UELException):
    pass
